tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
     "esModuleInterop": true,
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": true,
```

Comment:
Could you explain what is the reason for adding/removing these 2 rules?

Reply:
This is because importsNotUsedAsValues is deprecated, so I've switched to using the suggested verbatimModuleSyntax